src/DankCord/core.py

```diff
@@ -249,17 +295,34 @@ def crime(self, retry_attempts: int = 3, timeout: int = 10):
         
         Raises
         --------
+        InvalidComponent
+            invalidcomponentdescription
+        NonceTimeout
+            Couldn't get the nonce from the Websocket.
         UnknownChannel
             Bot doesn't have access to that channel.
 
         Returns
         --------
         message: Optional[`Message`]
         """
-        message: Optional[Message] = self.run_command("crime", retry_attempts, timeout)
-        return message
+        _location = None
+        if location not in [1, 2, 3, "random"]:
+            _location = "random"
+        if _location == "random":
+            _location = randint(1, 3)
+        else:
+            _location = location
```

Comment:
same goes here for the location code, and the docstring about the location arg, as well as renaming the arg